odiglet/pkg/instrumentation/plugin.go

```diff
@@ -70,7 +75,26 @@ func (p *plugin) Allocate(ctx context.Context, request *v1beta1.AllocateRequest)
 		}
 
 		deviceId := req.DevicesIDs[0]
-		res.ContainerResponses = append(res.ContainerResponses, p.LangSpecificFunc(deviceId))
+
+		cfg, err := rest.InClusterConfig()
```

Comment:
Maybe worth initializing kubernetes client in `NewPlugin` instead of in `Allocate` which is called every time we do instrumentation